perun-cli/setAttributeRights

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Getopt::Long qw(:config no_ignore_case);
+use Text::ASCIITable;
+use Perun::Agent;
+use Perun::Common qw(printMessage);
+#use Data::Dumper;
+
+sub help {
+	return qq{
+	Sets attribute rights of roles
+	--------------------------------
+	Available options:
+	--attributeId     | -a attribute identifier
+	--attributeName   | -A attribute name (including namespace)
+	--userRole        | -R user role (VOADMIN/GROUPADMIN/FACILITYADMIN/SELF)
+	--rolesRights     | -r list of rights for roles (READ WRITE)
+	--batch           | -b batch
+	--help            | -h prints this help
+	};
+}
+
+my ($attributeId, $attributeName, $reqRole, @rights, $batch);
+GetOptions("help|h"   => sub {
+		print help;
+		exit 0;
+	},
+	"attributeId|a=i"       => \$attributeId,
+	"attributeName|A=s"     => \$attributeName,
+	"userRole|R=s"          => \$reqRole,
+	'rolesRights|r=s@{1,}'  => \@rights,
```

Comment:
Right now tool can't remove attribute rights, since ``-r`` param is required to have value by the CLI, but empty list / null should be allowed in perun API. Please test it and allow removal either by another CLI tool or allowing empty list of rights.